services/ui-src/src/components/tables/Table.tsx

```diff
@@ -62,7 +62,7 @@ export const Table = ({
                   key={cell + index}
                   sx={{
                     tableCell: border ? sx.tableCellBorder : sx.tableCell,
-                    color: cell == "Not Answered" ? "palette.error_darker" : "",
+                    color: cell == "Not answered" ? "palette.error_darker" : "",
```

Comment:
Nit: Could use the constant notAnsweredText for this

Reply:
good idea! made this update, and in the export component as well